packages/react/src/module-federation/with-module-federation-ssr.ts

```diff
@@ -0,0 +1,64 @@
+import { ModuleFederationConfig } from '@nrwl/devkit';
+import { readCachedProjectConfiguration } from 'nx/src/project-graph/project-graph';
+import { getModuleFederationConfig } from './utils';
+
+function determineRemoteUrl(remote: string) {
+  const remoteConfiguration = readCachedProjectConfiguration(remote);
+  const serveTarget = remoteConfiguration?.targets?.serve;
+
+  if (!serveTarget) {
+    throw new Error(
+      `Cannot automatically determine URL of remote (${remote}). Looked for property "host" in the project's "serve" target.\n
+      You can also use the tuple syntax in your webpack config to configure your remotes. e.g. \`remotes: [['remote1', '//localhost:4201']]\``
+    );
+  }
+
+  const host = serveTarget.options?.host ?? '//localhost';
+  const port = serveTarget.options?.port ?? 4201;
+  return `${
+    host.endsWith('/') ? host.slice(0, -1) : host
+  }:${port}/remoteEntry.js`;
+}
```

Comment:
@jaysoo This logic may need to change depending on how the server is launched (if the path to the server's location of `remoteEntry.js` is not at the root URL for example).